CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 ## Unreleased
 
+## v0.16.0
+
+# 🎉 OpenTelemetry Collector Contrib v0.16.0 (Beta) 🎉
```

Comment:
Is there a guideline for what gets included here?

Reply:
@jpkrohling There is no official guideline, but I primarily refer to the changes since the previous release for components included the main executable that would affect the user experience.  Please feel free to provide any suggestions for anything that we should include/exclude.

Reply:
@jpkrohling if you want anything special to be included just make sure it is in the unreleased section so when we do the release we copy that

Reply:
> I primarily refer to the changes since the previous release for components included the main executable that would affect the user experience. 

Cool, that's pretty much what I had in mind as well :-) 

> if you want anything special to be included just make sure it is in the unreleased section so when we do the release we copy that

It was a genuine question about the release procedure and guidelines. Perhaps having a RELEASE.md describing how to release the project wouldn't be such a bad idea? If you like the idea, here are a couple of examples as reference:

https://github.com/jaegertracing/jaeger/blob/master/RELEASE.md
https://github.com/open-telemetry/opentelemetry-operator/blob/master/RELEASE.md

Reply:
@jparsana we have one https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/release.md
Feel free to suggest improvements.

Reply:
@jpkrohling we have one https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/release.md
Feel free to suggest improvements.

Reply:
Thanks, that's exactly what I was looking for, @tigrannajaryan!